lib/chain/src/ChainStore.js

```diff
@@ -4,6 +4,7 @@ import ChainTypes from "./ChainTypes";
 import ChainValidation from "./ChainValidation";
 import BigInteger from "bigi";
 import ee from "./EmitterInstance";
+import { asset } from "../../serializer/src/operations";
```

Comment:
this is ununused, please remove

Reply:
This is used here `    getAsset(id_or_symbol) {
        if (!id_or_symbol) return null;

        if (ChainValidation.is_object_id(id_or_symbol)) {
            let asset = this.getObject(id_or_symbol);

            if (
                asset &&
                (asset.get("bitasset") &&
                    !asset.getIn(["bitasset", "current_feed"]))
            ) {
                return undefined;
            }
            return asset;
        }

        /// TODO: verify id_or_symbol is a valid symbol name`

Reply:
You are right.